crates/algokit_transact/src/lib.rs

```diff
@@ -15,7 +15,7 @@ const ALGORAND_ADDRESS_LENGTH: usize = 58;
 const ALGORAND_PUBLIC_KEY_BYTE_LENGTH: usize = 32;
 
 #[derive(Debug, Error)]
-pub enum AlgoModelsError {
+pub enum AlgokitTransactError {
```

Comment:
```suggestion
pub enum AlgoKitTransactError {
```
We typically use `AlgoKit` in other repos.

Reply:
Nice catch. Force pushed [e10c345](https://github.com/algorandfoundation/algokit-core/pull/50/commits/e10c34522c51f4b15cc6884cd7a058b9b8934383) with the change Algokit -> AlgoKit

Reply:
Now [742ac11](https://github.com/algorandfoundation/algokit-core/pull/50/commits/742ac113c1b55b1bb2c5310f4ce433d7245069a7)